The pytest tests depend on a systemd environment
(systemd-hwdb is required).

--- a/meson.build
+++ b/meson.build
@@ -285,14 +285,6 @@ if get_option('tests').enabled()
 	else
 		message('valgrind not found, disabling valgrind test suite')
 	endif
-
-	pymod.find_installation(modules: ['libevdev', 'pyudev', 'pytest'])
-	pytest = find_program('pytest-3', 'pytest')
-	test('pytest',
-	     pytest,
-	     args: [meson.current_source_dir()],
-	     env: ['MESON_SOURCE_ROOT=@0@'.format(meson.current_source_dir())],
-	     suite: ['all'])
 endif
 
 # This is a non-optional test
